kube_metrics_server/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "display_name": "Kube_metrics_server",
```

Comment:
Can we change this to `Kube metrics server`? Its what will show up on the integration tile.

Reply:
Note this `ddev` PR helps do this for you in the future - https://github.com/DataDog/integrations-core/pull/3664/f

Reply:
Thank you for mentioning this :)